src/components/Dissolution/CompleteAffidavit.vue

```diff
@@ -123,6 +123,11 @@
             <span class="ml-2">Must be set to fit onto 8.5" x 11" letter-size paper
```

Comment:
Could you fix this while you're here? Span tags should be one the same line since spans are inline elements.

Rationale: With the closing tag on a separate line, there's actually a newline at the end of this text (even if it doesn't display).